xbmc/music/MusicDatabase.cpp

```diff
@@ -9479,12 +9482,13 @@ std::string CMusicDatabase::GetItemById(const std::string &itemType, int id)
   return "";
 }
 
-void CMusicDatabase::ExportToXML(const CLibExportSettings& settings,  CGUIDialogProgress* progressDialog /*= NULL*/)
+void CMusicDatabase::ExportToXML(const CLibExportSettings& settings,  CGUIDialogProgress *progressDialog /*= nullptr*/)
```

Comment:
left align the `*`, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
I swear we standarised on to the right, oh well so much for memory (and looking at too much old code in different style)